app/main/views/search.py

```diff
@@ -118,10 +118,11 @@ def delete_service(index_name, doc_type, service_id):
 
 
 def api_response(data, status_code, key='message'):
+    if status_code == 'N/A':
+        return jsonify(error=str(data)), 500
```

Comment:
How come we `str` this and not the others?

Reply:
`data` here is an exception, so it needs to get the string representation for serialization.

Reply:
👍  Yup :)